src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,18 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
```

Comment:
Don't extend your `Main/Application` class from `FigureSupplier` or `ColorSupplier`. Currently, it seems fine, but remember to keep the main class clean and not to inherit from utility classes.